docs/source-fabric/api/fabric_methods.rst

```diff
@@ -254,10 +254,30 @@ Fabric will handle the loading part correctly, whether running a single device,
 
     # Or load everything and restore your objects manually
     checkpoint = fabric.load("./checkpoints/version_2/checkpoint.ckpt")
-    model.load_state_dict(all_states["model"])
+    model.load_state_dict(checkpoint["model"])
     ...
 
 
+To load the state of your model or optimizer from a raw PyTorch checkpoint (not saved with Fabric), use :meth:`~lightning.fabric.fabric.Fabric.load_raw` instead.
+See also: :doc:`../guide/checkpoint`
+
+
+load_raw
+========
+
+Load the state-dict of a model or optimizer from a raw PyTorch checkpoint not saved by Fabric.
+
+.. code-block:: python
+
+    model = MyModel()
+
+    # A model weights file saved by your friend who doesn't use Fabric
+    fabric.load_raw("path/to/model.pt", model)
+
+    # Equivalent to this:
```

Comment:
maybe have the line below as a comment so not to confuse user if he needs to copy this too?